### ISSUE TITLE
Enable zooming

### ISSUE THREAD

**Issue by simov** [kdzwinel/betwixt#51]
:wave: is it possible to enable zooming like in regular Browser/Electron app? The regular shortcuts on Linux are not working: <kbd>ctrl</kbd> + <kbd>scroll up/down</kbd> and <kbd>ctrl</kbd> + <kbd>+/-</kbd>

**Comment by kdzwinel** [comment]
Good catch! I never explicitly disabled it, so I'm not sure why it's blocked - probably an Electron thing. If you can look it up and open a PR I'll be happy to merge.

BTW zoom gesture works on Mac, but it doesn't do the same thing as <kbd>ctrl</kbd> + <kbd>+/-</kbd>.
